packages/terra-application-layout/src/header/_ApplicationHeader.jsx

```diff
@@ -202,7 +202,7 @@ class ApplicationHeader extends React.Component {
     if (utilityComponent) {
       return (
         <Popup
-          attachmentBehavior="none"
+          attachmentBehavior="push"
```

Comment:
This change is worth pointing out. This should be passive for the ApplicationHeader due to being hardcoded.